### PR TITLE
Remove dev/staging from google

### PR DESCRIPTION
**Pull request by richtier**
[this task](https://uktrade.atlassian.net/browse/TT-160)

### CODE REVIEW

File: conf/settings.py [hunk 1]
```diff
@@ -51,7 +51,6 @@
     'django.contrib.messages',
     'raven.contrib.django.raven_compat',
     'django.contrib.sessions',
-    'config',
```
Comment (eadpearce) [inline conf/settings.py]: does this need to be 'conf' instead?

Comment (richtier) [reply conf/settings.py]: we never really needed this. the entries in `INSTALLED_APPS` help django collect static files, load models, locate templates, and locate template tags. As `conf/config` has none of those things we don't need it `INSTALLED_APPS`.

Comment (eadpearce) [reply conf/settings.py]: neat. the more u know

Comment (richtier) [reply conf/settings.py]: :rainbow:

**Pull request merged**